src/components/copy-link/CopyLink.js

```diff
@@ -42,6 +43,12 @@ class CopyLink extends React.Component {
             </div>
           </CopyToClipboard>
         </div>
+        <div className="overflow-hidden">
+          <div className="flex flex-column items-center appear-from-below">
+            <span className="charcoal f5 pb2">{t('copyLink.qrLabel')}</span>
```

Comment:
I suspect we want to make font size smaller for this label, perhaps the same as "This link changes.." ?

> ![2021-02-03--19-42-58](https://user-images.githubusercontent.com/157609/106793758-2642cf80-6658-11eb-8773-392f19d17e10.png)

Reply:
@rafaelramalho19 just put that there so I could style it. Watch this space.